src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,72 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Status } from '../../types/Status';
+
+type Props = {
+  onStatus: (status: Status) => void;
```

Comment:
the name does not say when the function will be called
therefore, it is better to rename it to `onChangeStatus`